doc/nrf/known_issues.rst

```diff
@@ -1124,6 +1124,13 @@ Subsystems
 Bluetooth® LE
 =============
 
+.. rst-class:: v2-0-0
+
+NCSDK-15724: Peripheral :ref:`peripheral_uart` sample fails to start on Thingy:53
+  Enabling USB by the sample's main function ends with error, because the USB was already enabled by Thingy:53-specific code.
```

Comment:
```suggestion
  Enabling USB by the :ref:`Peripheral UART's <peripheral_uart>` main function ends with error because the USB was already enabled by the Thingy:53-specific code.
```